torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -535,6 +540,36 @@ def _annotate_maxpool2d(
             _is_output_of_quantized_pattern=True,
         )
 
+    def _annotate_cat(
+        self, node: Node, quantization_config: QuantizationConfig
+    ) -> None:
+        cat_node = node
+        input_nodes = cat_node.args[0]
+        assert isinstance(input_nodes, Sequence)
+        first_input_node = input_nodes[0]
+        input_qspec_map = {}
+        assert isinstance(first_input_node, Node)
+        assert isinstance(cat_node, Node)
+        input_qspec_map[first_input_node] = get_input_act_qspec(quantization_config)
+        share_qparams_with_input_act0_qspec = SharedQuantizationSpec(
+            (first_input_node, cat_node)
+        )
+
+        for input_node in input_nodes[1:]:
+            if input_node not in input_qspec_map:
+                # There has the case of cat same nodes: torch.cat([input0, input0], 1)
+                assert isinstance(input_node, Node)
+                assert isinstance(
+                    share_qparams_with_input_act0_qspec, QuantizationSpecBase
+                )
```

Comment:
`share_qparams_with_input_act0_qspec` is constructed as `SharedQuantizationSpec`. This check seems redundant?

Reply:
I remember there are some format check error if we remove it previously. I will remove it and check the preCI again.